### ISSUE TITLE
[$40] NVDA Does Not Read Items On Page

### ISSUE THREAD

**Issue by hokienick** [topcoder-platform/community-app#3120]
A few instances of this occurring were reported in a previous bug hunt. These tickets have been condensed to this one ticket. The main idea for this ticket is that a user with a disability of sight should be able to use NVDA to read the items on the page.

In order to complete this ticket, make sure each of these tickets are satisfied:

#3104 - Footer Menu
#3103 - Header Menu / Sub-Menus
#3102 - Social Icons

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097703 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097703 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by kkartunov** [comment]
@topcoder-platform/topcodercompetitors this open for pickup

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097703 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by r0hit-gupta** [comment]
Footer Menu - Solved by PR https://github.com/topcoder-platform/community-app/pull/2992/files
Header Menu / Sub-Menus - Solved by PR https://github.com/topcoder-platform/community-app/pull/2977
Social Icons - Solved by PR https://github.com/topcoder-platform/community-app/pull/3138

**Comment by sachin-maheshwari** [comment]
Add label to Code Review Passed as  all three PRs PR #2992, PR #2977 and PR #3138 already merged in qa-accessibility branch

**Comment by drasticdpk** [comment]
Verified with windows/Chrome/NVDA

**Comment by lakshmiathreya** [comment]
All 3 Issues referenced here have been verified on Chrome and closed. Hence closing this issue.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097703<br/><br/>```This is an automated message for crazyk via Topcoder X```
